### PR TITLE
Zen UK IPv6 Instructions

### PR DESCRIPTION
**Pull request by marjohn56**
Zen UK IPv6 Instructions.

White Gfx

### CODE REVIEW

File: source/manual/IPv6_ZenUK.rst [hunk 1]
```diff
@@ -0,0 +1,212 @@
+**IPv6 For Zen UK**
+===================
+
+**Original Author:** Martin Wasley
+
+**Introduction**
+-----------------
+Zen provide two methods of setting up IPv6.
+
+The first method is a simple dhcp method which should suffice most
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: DHCP

File: source/manual/IPv6_ZenUK.rst [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+users, the second allows you to set up static IPv6 on both WAN and LAN.
+In either case the addresses and prefixes are constant and even under
+dhcp will not change. Currently Zen provide a /64 WAN address and a /48
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: DHCP

File: source/manual/IPv6_ZenUK.rst [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+to wait for an RA from the Zen gateway.
+
+The only other requirement in this section is to select ‘Use IPv4
+connectivity’, this is because the IPv6 traffic is routed over the PPPoE
+link.
+
+In the example above, ‘Enable debug’ is selected, this adds extra dhcp6c
+logging information to the dhcpd logs, this is optional.
+
+Click ‘Save’ and then ‘Apply’
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: dot missing

File: source/manual/IPv6_ZenUK.rst [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+As stated before, Zen provide a /48 prefix, so select the prefix size
+accordingly. We directly send the solicit as in this case we do not wish
+to wait for an RA from the Zen gateway.
+
+The only other requirement in this section is to select ‘Use IPv4
+connectivity’, this is because the IPv6 traffic is routed over the PPPoE
+link.
+
+In the example above, ‘Enable debug’ is selected, this adds extra dhcp6c
+logging information to the dhcpd logs, this is optional.
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: DHCP

File: source/manual/IPv6_ZenUK.rst [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+
+Click ‘Save’ and then ‘Apply’.
+
+It is advisable at this point to reboot the system.
+
+**Setting up IPv6 using Static Assignment**
+-------------------------------------------
+
+Although slightly more complex, this option gives you greater control
+over LAN dhcp6 server, as this can be tailored to specific needs.
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: DHCPv6

File: source/manual/IPv6_ZenUK.rst [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+
+We can take advantage of the link-local that is automatically set up
+between our router and the Zen BNG, we do not need to know specific
+gateways at the Zen end, it all gets routed automagically, however we do
+need to set up all of our static assignments, and our dhcpv6/RADVD
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: DHCPv6

File: source/manual/IPv6_ZenUK.rst [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+
+2a02:8231:d256:318::/66
+
+Pick an address, we’ll use ‘1’, so our WAN address
+2a02:8231:d256:318::1, it’s also going to be our gateway.
+
+We will also add a target IP for our monitor, the target will be a
+global WAN address of something. You can use the Google v6 DNS server
+address if you wish. We do NOT use the gateway address as our monitor,
+it’s pointless.
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: A bit harsh for technical documentation. How about something along the lines of:

If no monitor is given the gateway address is used, which can check gateway health status but may or may not indicate access to the Internet.

File: source/manual/IPv6_ZenUK.rst [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+
+Go to IPv6 Configuration Type and Select Static IPv6.
+
+.. image:: images/ZenUK_image5.png
+	:scale: 100%
+
+Go to Static IPv6 Configuration and set the IPv6 Static address:
+
+**Tip Use the same address as found when setting the system up to use
+dhcpv6.**
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: DHCPv6

File: source/manual/IPv6_ZenUK.rst [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+
+2a02:8242:55AB:0:4:3:2:1 would suffice.
+
+.. image:: images/ZenUK_image9.png
+	:scale: 100%
+
+We want to use a /64 prefix on this interface.
+
+**Tip Use the same address as found when setting the system up to use
+dhcpv6.**
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: DHCPv6

File: source/manual/IPv6_ZenUK.rst [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+
+Click Save and Apply.
+
+**DHCPv6 Server**
+-----------------
+
+When using DHCPv6 on the WAN, our dhcpv6 LAN server is set
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: DHCPv6

File: source/manual/IPv6_ZenUK.rst [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+
+Enter the upper – end range that the server will use.
+
+2a02:8231:d256::eeee:ffff:ffff:ffff
+
+.. image:: images/ZenUK_image11.png
+	:scale: 100%
+
+This should cover most LAN subnets, the range given here gives
+281,474.976.710,655 addresses!
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: Subtle irony of technical docs: "!" becomes "." :)

File: source/manual/IPv6_ZenUK.rst [hunk 12]
```diff
@@ -0,0 +1,212 @@
...
+sub routers or VLAN’s that need their own range. In the case of
+prefixes, we are only interested in the upper 64 bits, as in this
+example we will only be giving out 64 bit prefixes. We know we have been
+given a /48 prefix by Zen, so we enter our prefix range like this:
+
+.. image:: images/ZenUK_image12.png
+	:scale: 100%
+
+Our prefix range is the upper 48 bits, plus some of the next 16 bits,
+but we must NOT cross into the range we have used for our LAN addresses.
```
Comment (fichtner) [inline source/manual/IPv6_ZenUK.rst]: not

Special markup is bad for reading flow. People can either read or they can't.

**Comment by marjohn56** [comment]
That should do it.

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**